variants/frontend-bootstrap-typescript/app/frontend/stylesheets/customized_bootstrap.scss

```diff
@@ -0,0 +1,65 @@
+// Override Bootstrap's Sass default variables
+
+// Nearly all variables in Bootstrap are written with the `!default` flag.
+// This allows you to override the default values of those variables before
+// you import Bootstrap's source Sass files.
+
+// Overriding the default variable values is the best way to customize your
+// CSS without writing _new_ styles. For example, change you can either change
+// `$body-color` or write more CSS that override's Bootstrap's CSS like so:
+// `body { color: red; }`.
+
+@import "bootstrap/scss/functions"; // required
+@import "bootstrap/scss/mixins"; // required
```

Comment:
Shouldn't there be a `maps` import here now? I remember that being one of the fixes we applied recently to get the latest version of bootstrap working

Reply:
Correct, but actually this file should just be deleted because adding TypeScript doesn't need to change anything with `scss` files (which is why CI is passing: this file isn't being used by the new variant)